### PR TITLE
Update antimoderate license and keywords

### PR DESCRIPTION
**Pull request by programmiri**
Pull request for issue: 11931

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Squashing into only one commit now.

File: ajax/libs/antimoderate/package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "antimoderate",
   "version": "0.0.2",
-  "description": "The progressive image loading library for great good!",
-  "filename": "antimoderate.min.js",
+  "description": "The progressive image loading library",
+  "filename": "antimoderate.js",
```
Comment (sufuf3) [inline ajax/libs/antimoderate/package.json]: Let's use original `antimoderate.min.js`.
We also host `antimoderate.min.js` file, so we have to use `antimoderate.min.js` as filename.
Thanks.

File: ajax/libs/antimoderate/package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "antimoderate",
   "version": "0.0.2",
-  "description": "The progressive image loading library for great good!",
-  "filename": "antimoderate.min.js",
+  "description": "The progressive image loading library",
```
Comment (sufuf3) [inline ajax/libs/antimoderate/package.json]: It's fine to use origin description.
This is the same with https://github.com/whackashoe/antimoderate#antimoderate
Thanks.

File: ajax/libs/antimoderate/package.json [hunk 3]
```diff
@@ -16,15 +16,13 @@
...
-  },
+  "license": "MIT",
   "author": {
     "name": "Jett LaRue",
     "url": "https://jettlarue.com"
   },
   "keywords": [
-    "antimoderate"
+    "antimoderate",
+    "image"
```
Comment (PeterDaveHello) [inline ajax/libs/antimoderate/package.json]: Maybe add some more like `progressive`, `loading`?

File: ajax/libs/antimoderate/package.json [hunk 4]
```diff
@@ -16,15 +16,13 @@
       ]
     }
   ],
-  "license": {
-    "type": "See README.md",
-    "url": "https://github.com/whackashoe/antimoderate#license"
-  },
+  "license": "MIT",
```
Comment (PeterDaveHello) [inline ajax/libs/antimoderate/package.json]: From the readme, looks like `StackBlur Algorithm` is under MIT but its self may be CC-0 or public domain?

Comment (PeterDaveHello) [reply ajax/libs/antimoderate/package.json]: Though it is MIT in bower.json, but it'll be great that if you can open an issue ticket to confirm with its author about the license, would you?

File: ajax/libs/antimoderate/package.json (hunk 3 above)
Comment (PeterDaveHello) [reply ajax/libs/antimoderate/package.json]: PS: You can also contribute this back to its own bower.json in its repository, or even help it create a package.json if its author accepts.

File: ajax/libs/antimoderate/package.json (hunk 4 above)
Comment (programmiri) [reply ajax/libs/antimoderate/package.json]: Sure, I made an issue in the repo there to confirm the license -> https://github.com/whackashoe/antimoderate/issues/15

**Comment by maruilian11** [comment]
Hi @programmiri, Due to the author's reply, I think we can use `MIT`. :+1: 
Would you please help add more keywords, like `progressive` and `loading`? Thanks.
 https://github.com/cdnjs/cdnjs/pull/11989/commits/9c61f6f1467234ddac989fcc27a01fbc31ccf85b#r145030225

**Pull request merged**